Author: Agustin Martin <agmartin@debian.org>
Author: Axel Beckert <abe@debian.org>
Last-Update: 2020-05-03
Description: Add needed libs and filter out some unneeded, use pkg-config
Bug-Debian: https://bugs.debian.org/959646

--- a/configure.ac
+++ b/configure.ac
@@ -154,14 +154,14 @@
 fi
 
 dnl		WINGs support
-AC_CHECK_PROG(GETWINGSFLAGS,get-wings-flags,get-wings-flags,,[$PATH])
+AC_CHECK_PROG(GETWINGSFLAGS,pkg-config,pkg-config,,[$PATH])
 if test -z "$GETWINGSFLAGS"; then
-	AC_MSG_ERROR(['get-wings-flags' command was not found in \$PATH, please upgrade your WindowMaker.])
+	AC_MSG_ERROR(['pkg-config' command was not found in \$PATH, please install it.])
 fi
 
-WCFLAGS=`get-wings-flags --cflags`
-WLDFLAGS=`get-wings-flags --ldflags`
-WLIBS=`get-wings-flags --libs`
+WCFLAGS="`pkg-config --cflags WINGs` `pkg-config --cflags WUtil`"
+WLDFLAGS="`pkg-config --ldflags WINGs` `pkg-config --ldflags WUtil`"
+WLIBS="`pkg-config --libs WINGs` `pkg-config --libs WUtil`"
 
 AC_SUBST(XLIBS)
 AC_SUBST(XCFLAGS)
